superset-frontend/src/components/DeleteModal/index.tsx

```diff
@@ -70,8 +70,9 @@ export default function DeleteModal({
         </FormLabel>
         <FormControl
           data-test="delete-modal-input"
-          id="delete"
+          data-test-id="delete-modal-input"
```

Comment:
Superset already has a lot of these attrs as `data-test` (currently used for cypress e2e testing). Should we configure RTL to use this attr instead, so we don't end up with a bunch of duplicated attrs? 

We would just have to change this line, doesn't look like there are any other instances of `data-test-id` 
https://github.com/apache/superset/blob/a646914493c7bad943fcccf1985c6ee2d4d88269/superset-frontend/spec/helpers/setup.ts#L24

Reply:
cc @ktmud

Reply:
@nytai I agree. I didn't know it was configurable, you just changed my life kkkk

Done

Reply:
@nytai Yes, it should be configured as `data-test`. It was a mistake to use `data-test-id` when I set it up.